src/tests/mainTest.test.js

```diff
@@ -1,5 +1,7 @@
 import signUpTest from './user.test';
 import userLoginTest from './userLogin.test';
+import userProfileInformation from './getUserAnEditProfileInfo.test';
 
 signUpTest();
 userLoginTest();
+userProfileInformation();
```

Comment:
@EugeneMunya  why haven't you tested this endpoint as well?

`router.get('/user/profile', auth.auth, userProfile.getProfileInformation);`

<img width="647" alt="Screen Shot 2020-02-22 at 09 39 42" src="https://user-images.githubusercontent.com/48021034/75088559-ad865880-5557-11ea-9b27-84b506d771f9.png">

Reply:
I'm still working on it

Reply:
I managed to test everything, now you can check